mmv1/products/firebasehosting/api.yaml

```diff
@@ -64,4 +64,67 @@ objects:
       - !ruby/object:Api::Type::String
         name: defaultUrl
         output: true
-        description: The default URL for the site in the form of https://{name}.web.app
\ No newline at end of file
+        description: The default URL for the site in the form of https://{name}.web.app
+  - !ruby/object:Api::Resource
+    name: 'Channel'
+    min_version: beta
+    base_url: projects/{{project}}/sites/{{site_id}}/channels
+    self_link: sites/{{site_id}}/channels/{{channel_id}}
+    create_url: sites/{{site_id}}/channels?channelId={{channel_id}}
+    update_verb: :PATCH
```

Comment:
If this is updatable can you add a handwritten update test? Especially when we are using an update technique like PATCH+update_mask

Reply:
Done